shared_model/backend/protobuf/transaction_responses/proto_tx_response.hpp

```diff
@@ -51,7 +51,12 @@ namespace shared_model {
        */
       const ResponseVariantType &get() const override;
 
-      const ErrorMessageType &errorMessage() const override;
+      virtual const StatelessErrorOrFailedCommandNameType &
+      statelessErrorOrCommandName() const override;
+
+      virtual FailedCommandIndexType failedCommandIndex() const override;
```

Comment:
Maybe hash of the transaction is more appropriate here?

Reply:
The index can help during understanding, exactly which command failed in the given transaction, so this information is useful